service/resource/legacyv2/spec.go

```diff
@@ -9,7 +9,8 @@ const (
 	clusterVersionOutputKey = "ClusterVersionOutput"
 	workerRoleKey           = "WorkerRole"
 
-	cloudFormationTemplatesDirectory = "service/templates/cloudformation"
+	cloudFormationGuestTemplatesDirectory = "service/templates/cloudformation/guest"
+	cloudFormationHostTemplatesDirectory  = "service/templates/cloudformation/host"
```

Comment:
I like the separate directories for guest and host.

It just got me thinking that we'll need to version the cloud formation templates at some point. I think we can do that as part of the "update" changes rather than in the migration.

Reply:
Agree, makes more sense as part of update.